app/views/subscriptions/index.html.erb

```diff
@@ -14,6 +15,12 @@
   <tbody class="table__body">
     <%- @subscriptions.each do |subscription| %>
       <tr class="table__row">
+        <%= content_tag :td, class: 'table__cell' do %>
+          <%- if subscription.refreshable? %>
+            <%- status_info = subscription_status_info(subscription) %>
+            <i class="subscription__status <%= status_info[:class] %>" aria-label="<%= status_info[:label] %>" />
```

Comment:
I'm not fully convinced that colored dots are the right solution here. While we cover screenreaders with the aria-label, the color itself might be unclear to a user (let alone someone with some form of color-blindness).
Maybe we can use icons to indicate the special statuses (warning, error and not yet fetched)? Since we don't show anything for newsletters it would be consistent to render nothing if everything is ok